addons/knowledge/static/src/js/knowledge_field_html_injector.js

```diff
@@ -0,0 +1,274 @@
+/** @odoo-module */
+import Widget from 'web.Widget';
+import { ComponentWrapper, WidgetAdapterMixin } from 'web.OwlCompatibility';
+import { useService } from "@web/core/utils/hooks";
+import { TemplateToolbar, FileToolbar } from './knowledge_toolbars';
+import { UnremovableBehavior } from './knowledge_behaviors';
+const { Component } = owl;
+
+/**
+ * Component used to access the @see uiService
+ */
+class FieldHtmlInjectorComponent extends Component {
+    setup() {
+        this.uiService = useService('ui');
```

Comment:
In a certain future, when you will use components instead of widgets, you could avoid to trimballer this.uiService from one method to another.
One day